core/templates/issue_pages.html

```diff
@@ -81,16 +76,15 @@
                         <p><a href="{{url}}">View Issue</a></p>
                       </div>
                     {% endif %}
-                </div>
-            </div>
-            {% if forloop.counter|divisibleby:"4" %}
+                </figure>
+
+            <!-- {% if forloop.counter|divisibleby:"4" %}
```

Comment:
Why is this commented out? If it does need to be commented out, we should use django commenting to make sure the code does not execute. If it can be deleted it should be

Reply:
Should be deleted - thank you!